ci-operator/jobs/openshift/cincinnati/openshift-cincinnati-master-presubmits.yaml

```diff
@@ -18,6 +18,7 @@ presubmits:
       containers:
       - args:
         - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --image-mirror-push-secret=/etc/push-secret/.dockerconfigjson
```

Comment:
This job doesn't have the push mount
/hold

Reply:
`presubmits.yaml`: stupid me.